File: source codes/Sasha Angermueller Lab 4/GradeReport.cs
// Class:	CSE 1321L
// Section: 	08         
// Term:		Fall 2018  
// Instructor:	
// Name:		Sasha Angermueller
// Lab#:	    4

using System;

class GradeReport {
  public static void Main (string[] args) {
    int grade, switchvalue;
    Console.WriteLine ("What was your grade on the test?");
    grade = Convert.ToInt32(Console.ReadLine());
    if (grade > 99)
      {switchvalue = 1;}
    else if (grade/10 == 9)
      {switchvalue = 2;}
    else if (grade/10==8)
      {switchvalue = 3;}
    else if (grade/10==7)
      {switchvalue = 4;}
    else if (grade/10==6)
      {switchvalue = 5;}
    else
      {switchvalue = 6;}
    switch (switchvalue)
      {
        case 1:
          Console.WriteLine("That grade is a perfect score. Well done.");
          break;
        case 2:
          Console.WriteLine("That grade is well above average. Excellent work.");
          break;
        case 3:
          Console.WriteLine("That grade is above average. Nice job.");
          break;
        case 4:
          Console.WriteLine("That is average work.");
          break;
        case 5:
          Console.WriteLine("That grade is not good, you should seek help!");
          break;
        case 6:
          Console.WriteLine("That grade is not passing.");
          break;
        default:
          Console.WriteLine("Something went horriably, horriably wrong!");
          break;
      }
  }
}

File: School source codes/Lab 4/GradeReport.rb
puts "What was your grade on the test?"
grade = gets.chomp.to_i
case grade
  when 100
    puts "That grade is a perfect score. Well done."
  when 90..99
    puts "That grade is well above average. Excellent work."
  when 80..89
    puts "That grade is above average. Nice job."
  when 70..79
    puts "That is average work."
  when 60..69
    puts "That grade is not good, you should seek help!"
  when 0..50
    puts "That grade is not passing."
  else 
    puts "Something went horriably, horriably wrong! Or you entered an invalid grade."
end